doc/authorization-config.md

```diff
@@ -0,0 +1,385 @@
+## Configuring The Authorization Service
+
+The `authorization` section in a Trapperkeeper configuration file controls the
+logic that the `wrap-with-authorization-check` handler uses to authorize a 
+[Ring](https://github.com/ring-clojure/ring) request.  Here is one example of
+ an `authorization` section, using the
+[HOCON](https://github.com/typesafehub/config/blob/master/HOCON.md)
+configuration format:
+
+~~~~hocon
+authorization: {
+    version: 1
+    rules: [
+            {
+                match-request: {
+                    path: "/my_path"
+                    type: path
+                    method: get
+                }
+                allow: "*"
+                sort-order: 1
+                name: "my_path"
+            },
+            {
+                match-request: {
+                    path: "/my_other_path"
+                    type: path
+                }
+                allow-unauthenticated: true
+                sort-order: 2
+                name: "my_other_path"
```

Comment:
Good idea.  I altered one of the rules to have a space in the `name` in eef1f8e.